configuration/configuration.go

```diff
@@ -152,7 +156,7 @@ func (c *ConfigurationData) setConfigDefaults() {
 	//-----
 	// HTTP
 	//-----
-	c.v.SetDefault(varHTTPAddress, "0.0.0.0:8089")
+	c.v.SetDefault(varHTTPAddress, "0.0.0.0:8080")
```

Comment:
I switched the default address to 8089 to be able to run both WIT and Auth locally without changing the default configurations.